src/components/DataTable/Stories/PaginationTableStory.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { withInfo } from '@storybook/addon-info';
+import { MockPaginationTable } from '../__mocks__/mockPaginationTable';
+
+/**
+ * Pagination Table stories
+ */
+
+const paginationTableAddWithInfo = stories => {
+  stories.add(
+    'Paginated Table',
+    withInfo({
+      source: false,
+      propTablesExclude: [MockPaginationTable],
```

Comment:
Should we add  `propTables: [...]` to include the components that might be helpful to documented here and in the other examples?

Reply:
sure... will try to add some of them...

Reply:
So... I broke out all of the formatters into their own files/imports. I also elected to go ahead and them to the story sources as `propTables: [...]`, however the `tableCellFormatter` and `selectionCellFormatter` do not appear match the regexes checked by `react/no-unused-prop-types`. I think it is better to include them for documentation purposes and just eslint-ignore those checks for now.